README.md

```diff
@@ -54,6 +55,13 @@
     </tbody>
 </table>
 
+## Releases
+
+.NET for Apache Spark releases are available [here](https://github.com/dotnet/spark/releases).
+
+NuGet packages are available here:
+[![NuGet Badge](https://buildstats.info/nuget/Microsoft.Spark)](https://www.nuget.org/packages/Microsoft.Spark)
```

Comment:
@rapoth I need a suggestion on how to restructure/reword this paragraph/section.